ci/scripts/install_dask.sh

```diff
@@ -29,8 +29,7 @@ dask=$1
 if [ "${dask}" = "master" ]; then
   pip install https://github.com/dask/dask/archive/main.tar.gz#egg=dask[dataframe]
 elif [ "${dask}" = "latest" ]; then
-  conda install -q dask
+  pip install dask
```

Comment:
These scripts might be used from non-conda images so rather prefer pip here.

Reply:
This broke the nightly integration build, because the pip version not installing pandas automatically (now this is easily solved with `pip install dask[dataframe]`)

Reply:
Sorry, creating a PR for it.